tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -76,96 +78,139 @@ def base_setup(self, interface_type, pvc_size, pvc_factory, pod_factory):
             status=constants.STATUS_RUNNING
         )
 
+    def convert_str_size_to_int(self,str_size):
```

Comment:
@keesturam - Thank you for pointing this function. It does look similar. However, in this test this function cannot be used, since the units used for fillup_fs function are "M" and "G" and not "Mi" and "Gi". 
So it's either the convert_device_size should be adjusted for handling aditional ( 1 letter) unit formats or I should implement a function for handling G and M units, which I did. 
So can I leave it this way ?

Reply:
I think it will be better to change the parameter to use 'Mi' / 'Gi' so you can used the convert_device_size(), and the just remove the 'i' from the string when passing it to fillup_fs () function

Reply:
@Avilir , @keesturam - will do what Avi suggested.